web/public/js/services/cd-api.js

```diff
@@ -24,13 +24,13 @@ angular.module('cpZenPlatform').service('cdApi', ['$http', function($http) {
         });
       }, resolve, reject);
     },
-    get: function(url, resolve, reject) {
+    get: function(url, resolve, reject, params) {
```

Comment:
params here is more like additional options, right? We just happen to be using it to extend the object with a params property.